### DISCUSSION TITLE
How come there is no ComboBox control?

### DISCUSSION THREAD

**Discussion by jpgarza93** [lepoco/wpfui#219] [Q&A] [answered]
How come there is no ComboBox control? I downloaded the demo and could not see it anywhere

**Comment by pomianowski** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/13592821/173255442-22dc13f3-75a8-40fa-9e81-a77c0618e691.png)

**Reply by jpgarza93** [reply]
What sections is that on in the demo app? 
![image](https://user-images.githubusercontent.com/79015116/173447617-8cd3c0ce-6be7-4b99-a1ae-1ee575ad9661.png)

**Reply by jpgarza93** [reply]
I also can't seem to find it via the API:
![image](https://user-images.githubusercontent.com/79015116/173458741-c7d96785-db79-4f1d-abaf-a60bcf51e329.png)

**Reply by pomianowski** [reply]
You can just use build in WPF combo box via `<ComboBox/>`

**Reply by jpgarza93** [reply]
The problem is that I already have a default style for the ComboBox. I was thinking of using yours in specific places of my app.
